app-leechcraft/lc-liznoo/lc-liznoo-9999.ebuild

```diff
@@ -1,23 +1,26 @@
-# Copyright 1999-2018 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 inherit leechcraft
 
 DESCRIPTION="UPower-based power manager for LeechCraft"
 
 SLOT="0"
 KEYWORDS=""
-IUSE="debug"
+IUSE="+battery debug +poweractions +powerevents"
 
 DEPEND="~app-leechcraft/lc-core-${PV}
-	x11-libs/qwt:6
+	dev-qt/qtconcurrent:5
+	dev-qt/qtdbus:5
 	dev-qt/qtnetwork:5
 	dev-qt/qtwidgets:5
-	dev-qt/qtdbus:5
-	dev-qt/qtconcurrent:5
+	x11-libs/qwt:6
 "
 RDEPEND="${DEPEND}
 	~virtual/leechcraft-trayarea-${PV}
-	sys-power/upower"
+	battery? ( sys-power/upower )
+	poweractions? ( || ( sys-power/pm-utils sys-power/upower ) )
```

Comment:
at this point I'd say let's not introduce a new pm-utils consumer. it will be gone very soon.